src/validateUrl.js

```diff
@@ -0,0 +1,33 @@
+const { errorMessages } = require('./errorMessages');
+
+const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const validateUrl = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push(
+      {
+        message: errorMessages.MissingText,
+      },
+    );
+  }
+
+  if (!toCase) {
+    errors.push(
+      {
+        message: errorMessages.MissingCase,
+      },
+    );
+  } else if (!caseNames.includes(toCase)) {
+    errors.push(
+      {
+        message: errorMessages.WrongCase,
+      },
+    );
+  }
+
+  return errors;
```

Comment:
If this function validates data, maybe makes sense to throw a new error if validation failed? This function can return nothing, BUT throw error if array of errors is not empty. This behaviour is more understandable, than array of errors.

Reply:
![image](https://user-images.githubusercontent.com/121725856/235944607-f399a055-913a-4b76-9b6d-c89e0e8d08f5.png)
There is the requirement to red show errors like this to follow the payload, so I just think it may cause some tests problems.